egs/iwslt22_ta/ST/pruned_transducer_stateless5/train.py

```
@@ -0,0 +1,1290 @@
+    model: Union[nn.Module, DDP],
+    sp: spm.SentencePieceProcessor,
+    sp_tgt: spm.SentencePieceProcessor,
+    valid_dl: torch.utils.data.DataLoader,
+    world_size: int = 1,
+) -> MetricsTracker:
+    """Run the validation process."""
+    model.eval()
+
+    tot_loss = MetricsTracker()
+    with torch.no_grad():
+        for batch_idx, batch in enumerate(valid_dl):
```

Ruff (0.12.2)

Loop control variable `batch_idx` not used within loop body

Rename unused `batch_idx` to `_batch_idx`

(B007)